symfony/messenger/4.3/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#1": "For use AMQP install his pack by running composer require amqp",
+        "#2": "MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages",
+        "#3": "MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages",
+        "#4": "MESSENGER__FAILED_TRANSPORT_DSN=doctrine://default?queue_name=failed"
```

Comment:
If we remove this env var, this last example can go back to showing how to use doctrine for the main env var